runners/google-cloud-dataflow-java/src/main/java/org/apache/beam/runners/dataflow/options/DataflowPipelineOptions.java

```diff
@@ -53,7 +54,8 @@ public interface DataflowPipelineOptions
         StreamingOptions,
         DataflowWorkerLoggingOptions,
         DataflowProfilingOptions,
-        PubsubOptions {
+        PubsubOptions,
+        MemoryMonitorOptions {
```

Comment:
DataflowPipelineOptions extends DataflowPipelineDebugOptions extends DataflowPipelineDebugOptions. Is MemoryMonitorOptions also needed here?

Reply:
Good point, it was unnecessary indeed. I think I've done things in the wrong order and missed the chain